regression/cbmc-java/inherited_static_field5/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+test.class
+
+^VERIFICATION (SUCCESSFUL)|(FAILED)$
```

Comment:
If it must pass (as comment below suggests) how come we allow failed here? (Again the regex is wrong)